src/dmtcpplugin.cpp

```diff
@@ -199,7 +199,9 @@ EXTERNC int dmtcp_set_ckpt_dir(const char* dir)
 EXTERNC const char* dmtcp_get_coord_ckpt_dir(void)
 {
   static string dir;
+  dmtcp_disable_ckpt();
```

Comment:
You mean DMTCP_PLUGIN_DISABLE_CKPT()? I assume this is primarily used in plugins.